src/unittest/Dasm.cc

```diff
@@ -0,0 +1,55 @@
+#include "Dasm.hh"
+#include "EmuTime.hh"
+#include "MSXCPUInterface.hh"
+
+#include "catch.hpp"
+
+#include <iostream>
+
+using namespace openmsx;
+
+TEST_CASE("instructionLength, dasm")
+{
+	std::array<uint8_t, 4> opcode = {0, 0, 0, 0};
+	std::string dest;
+
+	bool end = false;
+	while (!end) {
+		auto len = instructionLength(opcode).value_or(0);
+		assert(1 <= len);
+		assert(len <= 4);
+
+		dest.clear();
+		auto dasmLength = dasm(std::span(opcode.data(), len), 0, dest);
+
+		CHECKED_ELSE( len == dasmLength ) {
+			std::stringstream ss;
+			ss << std::hex;
+
+			for (unsigned j = 0; j < opcode.size(); ++j) {
+				ss << std::setw(2) << std::setfill('0') << int(opcode[j]) << ' ';
+			}
+			ss << '(' << dest << ")\n";
+			INFO( "opcode: " << ss.str());
+
+			auto instructionLength = len;
```

Comment:
This line confused me for a moment. I guess it's a left-over from a previous version.
Just use `len` in the next line? Or on line 18 rename `len` to `instructionLength`?